content/Sleepawake.md

```diff
@@ -0,0 +1,33 @@
+---
+url: https://www.sleepawake.camp/about
+start: 
+end: 
+location: US
+target_age: 18-27
+image: /assets/sleepawake.png
+contributors: Matthew McCarthy
+---
+
+## Description
```

Comment:
Let's remove description heading. We can take it that first part is a short description of the organization.